src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Razor/GenerateMethodCodeActionResolver.cs

```diff
@@ -25,40 +25,31 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Razor;
 
-internal class GenerateMethodCodeActionResolver : IRazorCodeActionResolver
+internal sealed class GenerateMethodCodeActionResolver(
+    IDocumentContextFactory documentContextFactory,
+    RazorLSPOptionsMonitor razorLSPOptionsMonitor,
+    IClientConnection clientConnection,
+    IRazorDocumentMappingService razorDocumentMappingService,
+    IRazorFormattingService razorFormattingService) : IRazorCodeActionResolver
 {
-    private readonly IDocumentContextFactory _documentContextFactory;
-    private readonly RazorLSPOptionsMonitor _razorLSPOptionsMonitor;
-    private readonly IClientConnection _clientConnection;
-    private readonly IRazorDocumentMappingService _documentMappingService;
-    private readonly IRazorFormattingService _razorFormattingService;
-
+    private readonly IDocumentContextFactory _documentContextFactory = documentContextFactory;
+    private readonly RazorLSPOptionsMonitor _razorLSPOptionsMonitor = razorLSPOptionsMonitor;
+    private readonly IClientConnection _clientConnection = clientConnection;
+    private readonly IRazorDocumentMappingService _documentMappingService = razorDocumentMappingService;
+    private readonly IRazorFormattingService _razorFormattingService = razorFormattingService;
+
+    private const string ReturnType = "$$ReturnType$$";
+    private const string MethodName = "$$MethodName$$";
+    private const string EventArgs = "$$EventArgs$$";
     private static readonly string s_beginningIndents = $"{FormattingUtilities.InitialIndent}{FormattingUtilities.Indent}";
```

Comment:
Can `FormattingUtilities.InitialIndent` and `FormattingUtilities.Indent` be made const so that this can also be a const?

https://github.com/dotnet/razor/blob/b17a0e8ece87ee6c63527356a52e56db1ab5c6b6/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingUtilities.cs#L11-L12